clement-moulin-frier/vivarium
# Empty config list should hide corresponding config column

## Description
As @clement-moulin-frier asked, I made so that empty configs should not break the interface.

## Related Issue (if applicable)
#53

## How to Test
Launch the server
```
python3 scripts/run_server.py
```
Launch the Panel interface
```
panel serve scripts/run_interface.py --autoreload
```
No changes should be visible for now
